README.md

```diff
@@ -1,32 +1,302 @@
+operator?
+
+Arrays are an example of what is known in JavaScript as _reference_ types. This
+means that when you create an array and assign a label (a variable name) to it,
+that variable is pointing (_referring_) to a location in memory, not
+specifically to the value of the array. This can be contrasted with _primitive_
+types (strings, numbers, etc.), where the variable name points to a specific
+value.
+
+So what does that mean exactly? Let's look at an example. We'll create an array
+and then make a copy of it using the assignment operator:
+
+```js
+const myArray = ["one", "two", "three"];
+const myArrayCopy = myArray;
+```
+
+If we look at the value of our two arrays, as expected, they are the same:
+
+```js
+console.log(myArray);
+// => ["one", "two", "three"]
+console.log(myArrayCopy);
+// => ["one", "two", "three"]
+```
+
+But what if we make a change to the original array:
+
+```js
+myArray.push("four");
+```
+
+If we then look at the value of `myArray`, we'll see that "four" has been added
+to the end of the array, as expected:
+
+```js
+console.log(myArray);
+// => ["one", "two", "three", "four"];
+```
+
+But what about the value of `myArrayCopy`? Let's log that as well:
+
+```js
+console.log(myArrayCopy);
+// => ["one", "two", "three", "four"];
+```
+
+Is that what you were expecting to see?
+
+The reason this happened is because `myArray` is a reference pointing to our
+array in memory. When we created `myArrayCopy` and set it equal to `myArray`
+using the assignment operator, we created a new reference to _the same location
+in memory_. Then, when we modified `myArray`, we were changing the array stored
+in that location in memory. Since `myArrayCopy` is pointing to the same
+location, it reflects the change as well.
+
+<details><summary><b>what would you expect to happen if we modified `myArrayCopy` instead?</b></summary><p>
+  <ul>
+    <li>Exactly the same thing! Because both variables are pointing to the same location in memory, changing either one changes the other.</li>
+  </ul>
+</p></details>
+
+If we want to make a copy of an array that contains the same elements but
+doesn't point to the same location in memory as the original - i.e., a clone -
+we can instead use the spread operator:
+
+```js
+const myArray = ["one", "two", "three"];
+const clone = [...myArray];
+
+console.log(myArray);
+// => ["one", "two", "three"]
+console.log(clone);
+// => ["one", "two", "three"]
+```
+
+Now, if we change `myArray` and check the value of the two arrays:
+
+```js
+myArray.push("four");
+console.log(myArray);
+// => ["one", "two", "three", "four"]
+console.log(clone);
+// => ["one", "two", "three"]
+```
+
+The contents of our `clone` are unchanged!
+
+Note: can use `slice()` (without an argument) as an alternative to clone an
+array; it works the same way as the spread operator. Give it a try.
+
+## More Complex Modifications
+
+You may recall from the lesson on Array Methods in the Prep course that you can
```

Comment:
Not sure if you meant to link to the lesson here (Array Methods)

Reply:
Love the recall of previous material! Should help improve retention of previously taught content.